mirrors/index.rst

```diff
@@ -37,13 +37,15 @@ We thank the hosting institutions for mirroring the GMT data distribution so tha
 
 .. cssclass:: table-bordered
 
-==================== ============================================================= ========================
-Name                 Address                                                       Mirror Status
-==================== ============================================================= ========================
-**Oceania** [Master] https://oceania.generic-mapping-tools.org                     |Oceania_mirror_status|
-**Europe**           http://europe.generic-mapping-tools.org                       |Europe_mirror_status|
-**China**            http://china.generic-mapping-tools.org                        |China_mirror_status|
-==================== ============================================================= ========================
+==================== ============================================== ============================================= ========================
+Name                 Address                                        Host                                          Status
+==================== ============================================== ============================================= ========================
+**Oceania** [Master] https://oceania.generic-mapping-tools.org      SOEST, U of Hawaii, USA                       |Oceania_mirror_status|
+-------------------- ---------------------------------------------- --------------------------------------------- ------------------------
+**Europe**           http://europe.generic-mapping-tools.org        U of Algarve, Portugal                        |Europe_mirror_status|
+**China**            http://china.generic-mapping-tools.org         U of Science & Technology, China              |China_mirror_status|
+**South Africa**     https://south-africa.generic-mapping-tools.org TENET, Tertiary Education & Research Networks |SAfrica_mirror_status|
```

Comment:
It seems the South Afric mirror only supports HTTP, not HTTPS.
```suggestion
**South Africa**     http://south-africa.generic-mapping-tools.org TENET, Tertiary Education & Research Networks |SAfrica_mirror_status|
```

Reply:
Donald at TENET will add certificate once our end is set up, so we will leave it at https for now since that is what they will have.